back/engines/free/user_confirmation/app/controllers/user_confirmation/web_api/v1/confirmations_controller.rb

```diff
@@ -0,0 +1,25 @@
+module UserConfirmation
+  module WebApi
+    module V1
+      class ConfirmationsController < ApplicationController
+        skip_after_action :verify_authorized
+
+        def create
+          result = ConfirmUser.call(user: current_user, code: confirmation_params[:code])
+
+          if result.success?
+            head :ok
+          else
+            render json: { errors: result.errors.details }, status: :unprocessable_entity
+          end
+        end
```

Comment:
I like to keep controllers as lean as possible, with interactors and their business-oriented naming you can understand what the endpoint does by reading 5 lines.